plugins/azure/appConfigurations/appConfigurationManagedIdentity.js

```diff
@@ -0,0 +1,51 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'App Configurations Managed Identity',
+    category: 'App Configuration',
+    domain: 'Developer Tools',
+    description: 'Ensures that Azure App Configurations have managed identity enabled.',
+    more_info: 'Enabling managed identities eliminate the need for developers having to manage credentials by providing an identity for the Azure resource in Azure AD and using it to obtain Azure Active Directory (Azure AD) tokens.',
+    link: 'https://learn.microsoft.com/en-us/azure/azure-app-configuration/howto-integrate-azure-managed-service-identity?tabs=core6x&pivots=framework-dotnet',
+    recommended_action: 'Modify App Configuration store and add managed identity.',
+    apis: ['appConfigurations:list'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var locations = helpers.locations(settings.govcloud);
+
+        async.each(locations.appConfigurations, function(location, rcb){
+            var appConfigurations = helpers.addSource(cache, source, 
+                ['appConfigurations', 'list', location]);
+
+            if (!appConfigurations) return rcb();
+
+            if (appConfigurations.err || !appConfigurations.data) {
+                helpers.addResult(results, 3, 'Unable to query App Configuration: ' + helpers.addError(appConfigurations), location);
+                return rcb();
+            }
+
+            if (!appConfigurations.data.length) {
+                helpers.addResult(results, 0, 'No existing App Configurations found', location);
+                return rcb();
+            }
+
+            for (let appConfiguration of appConfigurations.data) {
+                if (!appConfiguration.id) continue;
+                var identityType = appConfiguration.identity && appConfiguration.identity.type? appConfiguration.identity.type : null;
+
+                if (identityType && (identityType.includes('systemassigned') || identityType.includes('userassigned'))) {
```

Comment:
```suggestion
                if (identityType && (identityType.toLowerCase().includes('systemassigned') || identityType.toLowerCase().includes('userassigned'))) {
```
is it necessary to check for identity type?